### PR TITLE
Bit 122 cli tests and debug

### PR DESCRIPTION
**Pull request by isabella618033**

### CODE REVIEW

File: bittensor/_executor/executor_impl.py [hunk 1]
```diff
@@ -193,6 +199,14 @@ def unstake_all ( self ):
 
     def unstake( self, amount_tao: int, uid: int ):
         r""" Unstaked token of amount to from uid.
+            Args:
```
Comment (unconst) [inline bittensor/_executor/executor_impl.py:197]: Nice commenting.

File: bittensor/_executor/executor_impl.py [hunk 2]
```diff
@@ -162,9 +163,13 @@ def overview ( self ):
         console = Console()
         console.print(table)
 
-
     def unstake_all ( self ):
-        r""" Unstaked from all hotkeys associated with this wallet's coldkey.
+        r""" Unstaked from all hotkeys associated with this wallet's coldkey. 
+            Args:
```
Comment (unconst) [inline bittensor/_executor/executor_impl.py]: This is nice, but if there are no args or return you don't need to add anything.
NIT no need for the extra lines.

File: bittensor/_executor/executor_impl.py [hunk 3]
```diff
@@ -193,6 +199,14 @@ def unstake_all ( self ):
 
     def unstake( self, amount_tao: int, uid: int ):
         r""" Unstaked token of amount to from uid.
+            Args:
+                amount_tao (int):
+                    ampunt of tao to be staked
+                uid (int):
+                    uid of the hotkey 
+
```
Comment (unconst) [inline bittensor/_executor/executor_impl.py]: Line not needed

File: bittensor/_executor/executor_impl.py [hunk 4]
```diff
@@ -275,8 +301,14 @@ def stake( self, amount_tao: int, uid: int ):
...
 
     def transfer( self, amount_tao: int, destination: str):
-        r""" Transfers token of amount to dest.
-            
+        r""" Transfers token of amount to destination.
+            Args:
+                amount_tao (int):
+                    ampunt of tao to be transfered
+                destination (str):
+                    destination public key address of reciever.
```
Comment (unconst) [inline bittensor/_executor/executor_impl.py]: Like this. :) perfect

File: bittensor/_executor/executor_impl.py [hunk 5]
```diff
@@ -275,8 +301,14 @@ def stake( self, amount_tao: int, uid: int ):
             logger.critical("Stake transaction failed")
 
     def transfer( self, amount_tao: int, destination: str):
-        r""" Transfers token of amount to dest.
-            
+        r""" Transfers token of amount to destination.
+            Args:
+                amount_tao (int):
+                    ampunt of tao to be transfered
```
Comment (unconst) [inline bittensor/_executor/executor_impl.py]: Capital letters and periods.

File: tests/unit_tests/bittensor_tests/test_executor.py [hunk 6]
```diff
@@ -93,7 +97,108 @@ def test_unstake_all_fail():
...
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        executor.unstake( amount_tao = 10, uid = uid-1)
+    
+    assert pytest_wrapped_e.type == SystemExit
+
+    # failing at subtensor add_stake
+    executor.subtensor.add_stake = MagicMock(return_value = False)
+    executor.unstake( amount_tao = 10, uid = uid)
+
+def test_stake():
```
Comment (unconst) [inline tests/unit_tests/bittensor_tests/test_executor.py:151]: really like this involved tests. Very thurough.

**Review by unconst — APPROVED** [review]

**Review by shibshib — APPROVED** [review]

File: bittensor/_executor/executor_impl.py (hunk 5 above)
Comment (unconst) [reply bittensor/_executor/executor_impl.py]: also spelling error

**Pull request merged**